apps/myaccount/src/components/shared/system-notification-alert.tsx

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Alert from "@oxygen-ui/react/Alert";
+import { ProfileConstants } from "@wso2is/core/constants";
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import {  LinkButton } from "@wso2is/react-components";
+import { AxiosError } from "axios";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch, useSelector } from "react-redux";
+import { Dispatch } from "redux";
+import { getPreference, resendVerificationLinkOrCode } from "../../api";
+import { ProfileConstants as MyAccountProfileConstants, RecoveryScenario } from "../../constants";
+import { SCIMConfigs } from "../../extensions/configs/scim";
+import { AlertLevels, AuthStateInterface } from "../../models";
+import { PreferenceConnectorResponse, PreferenceProperty, PreferenceRequest } from "../../models/preference";
+import { AppState } from "../../store";
+import { addAlert } from "../../store/actions";
+import "./system-notification-alert.scss";
+
+/**
+ * SystemNotificationAlert component props.
+ */
+export type SystemNotificationAlertProps = IdentifiableComponentInterface;
+
+/**
+ * System Notification Alert component.
+ *
+ * @param props - Props injected into the component.
+ * @returns ReactElement
+ */
+export const SystemNotificationAlert: FunctionComponent<SystemNotificationAlertProps> = (
+    props: SystemNotificationAlertProps
+): ReactElement | null => {
+    const { [ "data-componentid" ]: componentId } = props;
```

Comment:
For new components, lets destructure props inside the method signature itself.

```suggestion
export const SystemNotificationAlert: FunctionComponent<SystemNotificationAlertProps> = (
    { [ "data-componentid" ]: componentId }
): ReactElement | null => {
```